components/ServiceEdit/ServiceEdit.js

```diff
@@ -146,8 +146,8 @@ class ServiceEdit extends Component {
           {(taxonomies).map((category, i) => (
             <button
               key={`category-${i}`}
-              className={this.getCategoryClass(category)}
-              onClick={(e) => this.updateService('taxonomy', category.taxonomy.toLowerCase())}
+              className={this.getCategoryClass(category.name.toLowerCase())}
```

Comment:
They're getting mapped out to an object with `id`, `name`, and `icon`.  All we want is le name.